Initium-Odp/test/com/universeprojects/miniup/server/dao/CustomOrderDaoTest.java

```diff
@@ -0,0 +1,62 @@
+package com.universeprojects.miniup.server.dao;
```

Comment:
I don't think we should have a unit test for each kind. They should all be exactly the same and testing the same underlying stuff...